src/sentry/static/sentry/app/components/events/interfaces/debugMeta/debugFileDetails/index.tsx

```diff
@@ -133,6 +126,8 @@ const Value = styled(TextOverflow)<{coloredBg?: boolean}>`
   font-family: ${p => p.theme.text.familyMono};
 `;
 
+const ExternalDebugFilesTable = styled(Table)``;
```

Comment:
Is this necessary?

Reply:
it is not necessary, but because it is more descriptive, it may be easier to understand